### ISSUE TITLE
Context for ActivityStream responses

### ISSUE THREAD

**Issue by azaroth42** [linked-art/linked.art#527]
In #412 we decided on ActivityStreams as the response format when following HAL links (and also gets us aggregation inline with IIIF (and Europeana etc) in the future). That gives us the overall structure and syntax, but we still need to supply a context that can deal with both Linked Art classes and ActivityStream classes and properties. For example, the AS entry for a record about an object will have a class of `HumanMadeObject` rather than every record having a class of `Dataset` (meaning machine-readable content)

Options:

* Add ActivityStream classes into Linked Art context document with scoped contexts, just use the same context for all
* Create a second context only for the ActivityStreams content that includes the AS context and only the top level classes from Linked Art
* Have two contexts in the response, linked art and core ActivityStreams

My preference is for the first

**Comment by julsraemy** [comment]
I like the first option

**Comment by azaroth42** [comment]
In working on the issue here https://linked.art/api/1.0/ecosystem/search/ I went with the second option, due to collisions between the contexts. For example, both define `Person` (see https://www.w3.org/TR/activitystreams-vocabulary/#dfn-person), `content`, and `duration`. 

With two contexts we can manage collisions more easily. Also, to follow the IIIF usage, I defined `label` and `summary` as language maps, rather than requiring the more convoluted `nameMap` and `summaryMap` from the AS context.
